Kickstarter-iOS/Features/LoginTout/Controller/LoginToutViewController.swift

```diff
@@ -37,6 +37,10 @@ public final class LoginToutViewController: UIViewController, MFMailComposeViewC
     |> \.translatesAutoresizingMaskIntoConstraints .~ false
   }()
 
+  private lazy var loginOrSignUpWithOAuthButton = { UIButton(type: .custom)
```

Comment:
Most of this boiler-plate code is splitting out the "Sign up or Log in" button from the "Log In" button, where it was stuffed earlier.